develop/package_basics.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Creating a new R package for population genetics"
+---
+
+Motivation
+=========
+
+R packages are, at their core, standardized collections of functions, data sets,
+and/or data formats. The advantage to packages is that they can be used across
+all platforms, meaning that your analyses or visualizations can be easily shared
+among collaborators (including your future self). If you're reading this, then
+there is a good chance that you have written code that performs an analysis,
+does visualization, or helps you manage data. It is very likely that these tools
+you have authored will be of benefit to the larger population genetics
+community. This short vignette will point you to the tools you will need in
+order to get started on writing an R package for the population genetics
+community and submit it to [CRAN](http://cran.r-project.org).
+
+Tools/Resources
+=========
+
+Recently, many resources have emerged for developing R packages. We will use 
+Hadley Wickham's [R packages](http://r-pkgs.had.co.nz/) for reference, but it's 
+valuable to check out other tutorials.
+
+## Tools
+
+Writing R packages has become much easier with the advent of the IDE
+[Rstudio](http://rstudio.org) and the package
+[devtools](https://github.com/devtools). Using both of these in conjunction will
+make developing your package much easier as they are designed to work together.
+
+- [Download Rstudio here](http://rstudio.org/download)
+
+You can install `devtools` by typing in your R console:
+
+```{r, devtools_install, eval = FALSE}
+install.packages("devtools", repos = "http://cran.rstudio.com/")
+```
+
+You might also need some background noise to avoid distraction. I recommend 
+[the sound of a space ship control room](http://mynoise.net/NoiseMachines/spaceshipNoiseGenerator.php).
+
+## Tutorials
+
+- [Package Structure (Wickham)](http://r-pkgs.had.co.nz/package.html)
+- [Writing an R package from scratch (cat themed) (Parker)](http://hilaryparker.com/2014/04/29/writing-an-r-package-from-scratch/)
+- [Developing R packages (Leek)](https://github.com/jtleek/rpackages)
+- [Creating Packages (Leisch)](http://cran.r-project.org/doc/contrib/Leisch-CreatingPackages.pdf)
+- [Writing R Extensions (CRAN)](http://cran.r-project.org/doc/manuals/r-release/R-exts.html)
```

Comment:
I would spell out attribution to full names, and move them to after (rather than part of) the hyperlinked text. (But it's a matter of taste also, so feel free to refuse :smile:)